homeassistant/components/switch/rest.py

```diff
@@ -36,26 +37,38 @@ def setup_platform(hass, config, add_devices_callback, discovery_info=None):
         _LOGGER.error("No route to resource/endpoint: %s", resource)
         return False
 
+    cfg_headers = config.get('headers')
```

Comment:
You just want to get the headers key, and if it's not there you want to use the default vlaue:

```python
cfg_headers = config.get('headers', DEFAULT_HEADERS)
```